client/gutenberg/extensions/wordads/editor.scss

```diff
@@ -1,6 +1,13 @@
-.aligncenter .wp-block-jetpack-wordads {
-justify-content:center;
+.wp-block-jetpack-wordads {
+	background: #FFF;
```

Comment:
We should prolly use `$white` sass variable here, or at least lowercase `#fff` to please `stylelint`.